src/pharmpy/tools/covsearch/tool.py

```diff
@@ -441,21 +457,22 @@ def task_results(p_forward: float, p_backward: float, state: SearchState):
         steps=steps,
         candidate_summary=candidate_summary_dataframe(steps),
         ofv_summary=ofv_summary_dataframe(steps, final_included=True, iterations=True),
-        summary_tool=_modify_summary_tool(res.summary_tool, steps),
+        summary_tool=_modify_summary_tool(res.summary_tool, steps, is_strict),
         summary_models=_summarize_models(models, steps),
     )
 
     return res
 
 
-def _modify_summary_tool(summary_tool, steps):
+def _modify_summary_tool(summary_tool, steps, is_strict):
```

Comment:
To me `is_strict` is a bit unclear what it means, and since it's only used in this function I think it might be clearer to send in `strictness` and then set it directly when creating the `strictness_fulfilled` column.

Reply:
`is_strict` is a list of booleans. The entries are True if the models pass the strictness criteria, otherwise they are False. I cannot send in `strictness` directly to `_modify_summary_tool` because it is only a string and the models are needed to evaluate it.